src/main/target/common_pre.h

```diff
@@ -82,6 +83,7 @@
 #define USE_DSHOT_TELEMETRY
 #define USE_DSHOT_TELEMETRY_STATS
 #define USE_RPM_FILTER
+#define USE_DYN_IDLE
```

Comment:
You'll need to add:
```
#ifndef USE_RPM_FILTER
#undef USE_DYN_IDLE
#endif
```
to `target/common_post.h` to account for the dependency on RPM Filter.

Reply:
Ok